includes/polling/os/pfsense.inc.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+use LibreNMS\RRD\RrdDefinition;
+
+$oids = snmp_get_multi($device, 'pfStateTableCount.0 pfStateTableSearches.0 pfStateTableInserts.0 pfStateTableRemovals.0 pfCounterMatch.0 pfCounterBadOffset.0 pfCounterFragment.0 pfCounterShort.0 pfCounterNormalize.0 pfCounterMemDrop.0', '-OQUs', 'BEGEMOT-PF-MIB');
+
+$states = $oids[0]['pfStateTableCount'];
+$searches = $oids[0]['pfStateTableSearches'];
+$inserts = $oids[0]['pfStateTableInserts'];
+$removals = $oids[0]['pfStateTableCount'];
+$matches = $oids[0]['pfCounterMatch'];
+$badoffset = $oids[0]['pfCounterBadOffset'];
+$fragmented = $oids[0]['pfCounterFragment'];
+$short = $oids[0]['pfCounterShort'];
+$normalized = $oids[0]['pfCounterNormalize'];
+$memdropped = $oids[0]['pfCounterMemDrop'];
+
+
+if (is_numeric($states)) {
```

Comment:
Not really sure if this will be a problem or not but as you check if each of these values are numeric then it could mean that in the event they aren't then from one poll to another you could end up with data out of order or the wrong number of DS' in the RRD if you have transient issues with the `snmp_multi_get()` call.

Reply:
@laf I don't think that is an issue here since the data is fetched by oid name.

Reply:
Actually I've just realised that these are all creating separate files anyway so this isn't an issue.

Reply:
If you can fix the mib issues from my comments I think we're good to go.